test/integration_test.js

```diff
@@ -167,10 +167,8 @@ describe("Integration tests", () => {
           expect(cleanRecord(syncResult.published[0])).eql(testData.localUnsynced[0]);
         });
 
-        it("should mark local records as synced", () => {
```

Comment:
Are we still testing that it does actually mark the local record as synced? I guess we were testing that implicitly through the inclusion in`updated`, and now we're not testing it at all anymore? (at least not here).